mmv1/products/looker/Instance.yaml

```diff
@@ -416,3 +424,34 @@ properties:
         description: |
           Number of additional Developer Users to allocate to the Looker Instance.
   # UserMetadata Object - End
+  # CustomDomain Object
+  - !ruby/object:Api::Type::NestedObject
+    name: customDomain
+    description: |
+      Custom domain settings for a Looker instance.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'domain'
+        description: |
+          Domain name
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        output: true
+        description: |
```

Comment:
For output-only enums, we generally use a string instead, because validation is not needed. You can also simplify the description to just `Status of the custom domain`, which is nice in case the options change in the future.

Reply:
All right, yeah makes sense. I can definitely change it. I kept it the same as the other output enums in the Looker resource, like kmsKeyState. But simplifying it sounds like a good idea.